src/utils/middleware/lollipop.ts

```diff
@@ -19,12 +21,28 @@ import {
   withLollipopHeadersFromRequest
 } from "../../types/lollipop";
 import { log } from "../logger";
+import { LollipopSignatureInput } from "../../../generated/lollipop/LollipopSignatureInput";
 
 type ErrorsResponses =
   | IResponseErrorInternal
   | IResponseErrorForbiddenNotAuthorized;
 
-export const lollipopMiddleware: (
+const LOLLIPOP_SIGN_ERROR_EVENT_NAME = "lollipop.error.sign";
```

Comment:
maybe we can refactor it with a function to manage the whole object
```
{
    assertion_ref: assertionRef,
    fiscal_code: sha256(user.fiscal_code),
    name: LOLLIPOP_SIGN_ERROR_EVENT_NAME,
    operation_id: operationId
}
```

Reply:
IMHO the change make the `expressLollipopMiddleware` more complex because we should carry across all the pipe the initialized function.